### DISCUSSION TITLE
change default user

### DISCUSSION THREAD

**Discussion by Tealk** [FreshRSS/FreshRSS#4430] [Q&A] [answered]
Hello,

I find the idea of anonymous reading of the default user interesting: "Allow anonymous reading of articles of the default user (Tealk)", but can I change which user it should be without changing the administrator?

**Comment by Alkarex** [comment, marked as answer]
Hello,
The default user can be changed in `./data/config.php` with the field `default_user`.
The information about being admin or not is now stored individually for each user.

**Reply by Tealk** [reply]
So the comment is no longer correct?
```
# Name of the user that has administration rights.
default_user' => '_',
```

**Reply by Alkarex** [reply]
Indeed. PR welcome

**Reply by Alkarex** [reply]
https://github.com/FreshRSS/FreshRSS/pull/4434

**Comment by math-GH** [comment]
Here you can find information how to change the default user: https://freshrss.github.io/FreshRSS/en/admins/13_Default_user.html
